core/src/test/java/io/grpc/internal/ServiceConfigInterceptorTest.java

```diff
@@ -98,6 +100,16 @@ public void withWaitForReady() {
     assertThat(callOptionsCap.getValue().isWaitForReady()).isTrue();
   }
 
+  @Test
+  public void notYetHandleUpdate() {
```

Comment:
This test name isn't clear. I think it's intended to test that the default retry policy is returned if `interceptor.handleUpdate` hasn't been called, but there's no accompanying test for when `handleUpdate` has been called - this functionality seems covered in the `RetryPolicyTest` cases, but if it's going to be tested here as well (which makes sense) we should also cover the case when it is actually updated.